frame/system/benchmarking/src/lib.rs

```diff
@@ -18,6 +18,7 @@
 // Benchmarks for Utility Pallet
 
 #![cfg_attr(not(feature = "std"), no_std)]
+#![cfg(feature = "runtime-benchmarks")]
```

Comment:
Weird that the pallet even has this feature as this pallet should only be used when `runtime-benchmarks` are enabled somewhere else.

Reply:
I was just running the tests for all pallets and one of the benchmarking tests failed. So it was "enabled" manually by `cargo test`.

Reply:
without this, my build isn't working. can we have this ported to the 1.0.0 release branch @bkchr

Reply:
this is the correct solution https://github.com/paritytech/substrate/pull/14617

Reply:
We need this for Acala to compile with polkadot-v1.0.0 branch